cli/cmd/lql_sources.go

```diff
@@ -65,6 +66,11 @@ func getListQuerySourcesTable(datasources []api.Datasource) (out [][]string) {
 func listQuerySources(_ *cobra.Command, args []string) error {
 	cli.Log.Debugw("retrieving LQL data sources")
 
+	tableSettings := tableFunc(func(t *tablewriter.Table) {
+		t.SetAutoWrapText(false)
+		t.SetBorder(false)
+	})
+
```

Comment:
Why do we define these settings all the way up here? Why not just nested right when we use them below?